toolkit/tools/imagecustomizer/docs/configuration.md

```diff
@@ -36,7 +35,9 @@ The Mariner Image Customizer is configured using a YAML (or JSON) file.
 
 10. Run post-install scripts. ([PostInstallScripts](#postinstallscripts-script))
 
-11. Apply kernel command-line args, if the partitions weren't customized.
+11. If [ResetBootLoaderType](#resetbootloadertype-string) is specified, then reset the
+    boot-loader.
+    Otherwise, append kernel command-line args to existing boot-loader config.
```

Comment:
Not clear if that appends the kernel cmdline from the input images or from the config or both? 
Also maybe call out explicitly that this is the default behavior in the paritions API section?

Reply:
Reading the rest of the PR, you have it explained wel later. Though it looks like ResetBootLoaderType is mandatory so you may want to reword this a bit to call out "ResetBootLoaderType is set to `hard-reset` instead of the "If" statement

Reply:
It is only mandatory if partitions are customized.